mgmtsystem/__manifest__.py

```diff
@@ -39,5 +23,5 @@
         'views/res_config.xml'
     ],
     "demo": [],
-    'installable': False,
+    'installable': True,
 }
```

Comment:
document_page should be a dependency of mgmtsystem_manual, _procedure and _work_instruction